.github/workflows/release.yml

```diff
@@ -235,25 +241,33 @@ jobs:
         with:
           name: deb
 
+      - uses: docker/setup-qemu-action@v1
+
       - uses: docker/setup-buildx-action@v1
 
       - name: generate dockerfile
         run: |
           echo 'FROM ${{ matrix.distro }}
-          COPY tyk-pump*_amd64.deb /tyk-pump.deb
+          ARG TARGETARCH
+          COPY tyk-pump*_${TARGETARCH}.deb /tyk-pump.deb
+          RUN apt-get update && apt-get install -y curl
+
+          RUN curl -fsSL https://packagecloud.io/install/repositories/tyk/tyk-pump/script.deb.sh | bash && apt-get install -y tyk-pump=1.2.0
           RUN dpkg -i /tyk-pump.deb && /opt/tyk-pump/tyk-pump --conf=/opt/tyk-pump/tyk-pump.conf &' > Dockerfile
```

Comment:
Does it actually checks if pump run successfully? 
Can we have some simple "custom" tyk-pump.conf, with "CSV" pump or similar, and after running pump, check its STDOUT to ensure that old config was loaded (check for CSV keyword for example, or lack of mongo keyword)

Reply:
Yes, this will be a better test. I do not have a known good config file nor am I clear on how it can be tested. @ilijabojanovic could you help me with a better test case?

Reply:
Here is the config you can use for pump:

```
{
    "analytics_storage_type": "redis",
    "analytics_storage_config": {
        "type": "redis",
        "host": "localhost",
        "port": 6379,
        "username": "",
        "password": "",
        "database": 0,
        "optimisation_max_idle": 500,
        "optimisation_max_active": 1000
    },
    "purge_delay": 5,
    "dont_purge_uptime_data": true,
    "pumps": {
      "csv": {
        "type": "csv",
        "meta": {
          "csv_dir": "./"
        }
      }
    }
}
```

In STDOUT you should check for `Init Pump: CSV Pump` message.

Reply:
Only redis is dependency, mongo is not required.

Reply:
This sort of test is not easy to "templatetise" into https://github.com/TykTechnologies/tyk-ci/blob/master/wf-gen/release/install.m4#L39 for all the repos. I will think about how this could be best implemented.

Alternatively, a pump is running in the dev envs and is updated for every commit. It would make sense to run a wider set of tests against this.

Reply:
Unlikely you can have "generalized" template which fits all repos. But you can add different actions based on repo name. E.g. different upgrade tests.

Reply:
Indeed. This is a problem that needs to be solved for the plugin tests and implementing it is more complex which is why I have not done it in this PR. Do you want to keep this PR open until I work all of this out?